File: src/net/eduard/api/command/api/ApiLoadCommand.java

package net.eduard.api.command.api;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class ApiLoadCommand extends CommandManager {

	public ApiLoadCommand() {
		super("load","carregar");
		setUsage("/api load <plugin>");
		setDescription("Carrega um plugin descarregado no servidor");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command,
			String label, String[] args) {
		sender.sendMessage("§bEduardAPI §cNão implementado");
		
		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiLoadCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiLoadCommand : CommandManager("load", "carregar") {
    override fun onCommand(
        sender: CommandSender, command: Command,
        label: String, args: Array<String>
    ): Boolean {
        sender.sendMessage("§bEduardAPI §cNão implementado")
        return true
    }

    init {
        usage = "/api load <plugin>"
        description = "Carrega um plugin descarregado no servidor"
    }
}